providers/azure/aci.go

```diff
@@ -192,6 +206,21 @@ func NewACIProvider(config string, rm *manager.ResourceManager, nodeName, operat
 		p.pods = podsQuota
 	}
 
+	p.subnetName = "virtual-kubelet-aci-connector"
```

Comment:
remove the default value. If it's not set, the customer doesn't want to use VNet. you can skip setupNetworkProfile, then